apps/quickstart-react/package.json

```diff
@@ -15,8 +15,9 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
-    "monday-sdk-js": "^0.1.2",
-    "monday-ui-react-core": "0.0.48",
+    "kill-port": "^1.6.1",
```

Comment:
why do we need this?

Reply:
It's a command used in the start script. It is already installed on some people’s computers but we want to include it as an explicit dependency so that it doesn't fail for the other customers.